docs/blog/2023/announce-mystjs.md

```diff
@@ -0,0 +1,143 @@
+The project is called [MyST-JS](https://myst-tools.org/docs/mystjs).
+
+To drive this effort forward, we have [partnered with Curvenote](https://curvenote.com) to upstream their open-source documentation engine into the Executable Books community.
+This effort is based [on their command line interface](https://curvenote.com/docs/cli), which was designed for building technical documentation and computational narratives with JavaScript.
+It will become the backbone of **MyST-JS**, and moving forward will be maintained by the Executable Books community.
+There is still much to do, but you can preview this work already at [js.myst-tools.org](https://myst-tools.org/docs/mystjs).
+
+Here are a few major new steps summarizing this work over the coming months:
+
+### MyST is now a top-level project in Executable Books
+
+We'll treat MyST as another major effort of the Executable Books project, similar to Jupyter Book.
+It will have its own strategy and goals.
+Jupyter Book will remain a consumer of MyST markdown, and it will remain one of many stakeholders in the MyST ecosystem.
+
+:::{admonition} A new website!
+We have set up a dedicated site to describe the MyST project at [myst-tools.org](https://myst-tools.org).
+:::
+
+### MyST will have first-class support in JavaScript
+
+The [MyST-JS project](https://github.com/executablebooks/mystjs) is written entirelly in TypeScript, with the goal of being re-usable across more applications and interfaces.
+It also allows us to leverage the vast ecosystem of JavaScript libraries and frameworks for communication.
+For example, we can use [templating frameworks to export MyST documents to a variety of LaTeX publication formats](https://github.com/myst-templates).
+In the future, we hope to leverage the [unifiedjs project](https://unifiedjs.com/) to make MyST documents even more structured and consumable as data.
+
+:::{admonition} Learn more
+You can find the documentation for MyST JS at [js.myst-tools.org](https://myst-tools.org/docs/mystjs). You can also find an [issue from Rowan discussing major new planned functionality here](https://github.com/executablebooks/meta/issues/838).
+:::
+
+### MyST will have a Jupyter Lab extension
+
+By bringing MyST into JavaScript, we can integrate it directly with JupyterLab and other web-based Jupyter interfaces.
+We have already explored a few ways of doing this via JupyterLab extensions.
+The [`jupyterlab-myst` repository](https://github.com/executablebooks/jupyterlab-myst) brings basic functionality of MyST in JupyterLab via `markdown-it` plugins, and the [`jupyterlab-mystjs`](https://github.com/executablebooks/jupyterlab-mystjs) repository is a more fully-functional document engine with MyST in JupyterLab.
+We hope to merge the second repository into the first in the future, so follow [`jupyterlab-myst`](https://github.com/executablebooks/jupyterlab-myst) moving forward.
```

Comment:
@agoose77 and I have worked today to get these two repositories merged together, we are tracking all outstanding pieces in https://github.com/executablebooks/jupyterlab-myst/issues/57 and will have this fully deployed in the next day or so!

```suggestion
By bringing MyST into JavaScript, we can integrate it directly with JupyterLab and other web-based Jupyter interfaces.
For example, the [`jupyterlab-myst`](https://github.com/executablebooks/jupyterlab-myst) extension brings native MyST rendering into JupyterLab including admonitions, cross-references, and figure numbering using `mystjs` as the document engine.
```